src/test/java/org/checkstyle/autofix/recipe/RemoveViolationCommentsTest.java

```diff
@@ -28,7 +28,7 @@
 
 import com.puppycrawl.tools.checkstyle.AbstractPathTestSupport;
 
-public class RemoveViolationCommentsTest extends AbstractPathTestSupport implements RewriteTest {
+class RemoveViolationCommentsTest extends AbstractPathTestSupport implements RewriteTest {
```

Comment:
@timtebeek , why public is not recommended ?

Reply:
It's not necessary with Junit 5+, and it may lead to reuse of logic between packages, which we'd want to avoid.

Reply:
Reverted and disabled.